nbconvert/exporters/exporter.py

```diff
@@ -203,13 +203,13 @@ def register_preprocessor(self, preprocessor, enabled=False):
         """
         Register a preprocessor.
         Preprocessors are classes that act upon the notebook before it is
-        passed into the Jinja templating engine.  preprocessors are also
+        passed into the Jinja templating engine. Preprocessors are also
         capable of passing additional information to the Jinja
         templating engine.
 
         Parameters
         ----------
-        preprocessor : `Preprocessor`
+        preprocessor : `nbconvert.preprocessors.Preprocessor`
```

Comment:
i am not sure if this is the correct preprocessor object. It is either this or `.base.Preprocessor`. but this was causing the docs to fail too.